### ISSUE TITLE
Support \x for Expanded Display

### ISSUE THREAD

**Issue by jackmaney** [amjith/pgcli#19]
The `tabulate` output looks very pretty, but a feature I love from `psql` is the expanded display option (`\x`) for looking at rows in really wide tables (ie tables with rows that are too big to fit in one line on the terminal). However, this isn't yet supported in pgcli:

```
$ pgcli
[dbname]> \x
syntax error at or near "\"
LINE 1: \x
        ^
```

**Comment by jaytaylor** [comment]
+1

**Comment by amjith** [comment]
This was requested by a few people. Need to look at how to do this effectively.

**Comment by treystout** [comment]
+1

**Comment by cezio** [comment]
+1

**Comment by stedi67** [comment]
+1

**Comment by binarysource** [comment]
+1

**Comment by amjith** [comment]
Merged in PR #63 that adds \x support to master. 

This currently fails on Python 3, but that will be fixed before the next release. 

Can someone install the latest master and try it out? 

You can install the latest master using the following command. 

`pip install -e git+git@github.com:amjith/pgcli.git#egg=pgcli`

**Comment by jackmaney** [comment]
It mostly works, but it seems to be printing extra information:

```
$ pgcli
[dbname]> \x
Expanded display is on.
Expanded display is on.
[dbname]> select generate_series(1,5);
-[ RECORD 0 ]---+--
generate_series | 1
-[ RECORD 1 ]---+--
generate_series | 2
-[ RECORD 2 ]---+--
generate_series | 3
-[ RECORD 3 ]---+--
generate_series | 4
-[ RECORD 4 ]---+--
generate_series | 5

SELECT 5
+-------------------+
|   generate_series |
|-------------------|
|                 1 |
|                 2 |
|                 3 |
|                 4 |
|                 5 |
+-------------------+
SELECT 5
```

**Comment by jackmaney** [comment]
Whoah...it's not just with `\x`. I'm seeing double (`blarg` is just a test table):

```
$ pgcli
[dbname]> select * from blarg;
+-----------+
|         x |
|-----------|
|         2 |
|         3 |
| -17171717 |
+-----------+
SELECT 3
+-----------+
|         x |
|-----------|
|         2 |
|         3 |
| -17171717 |
+-----------+
SELECT 3
```

**Comment by jaytaylor** [comment]
Awesome work everybody!

**Comment by amjith** [comment]
This is now available in the release version 0.11.0

https://pypi.python.org/pypi?:action=display&name=pgcli&version=0.11.0
